Documentation/BasicUsage.md

```diff
@@ -155,8 +155,8 @@ peerConnection.addEventListener( 'iceconnectionstatechange', event => {} );
 peerConnection.addEventListener( 'icegatheringstatechange', event => {} );
 peerConnection.addEventListener( 'negotiationneeded', event => {} );
 peerConnection.addEventListener( 'signalingstatechange', event => {} );
-peerConnection.addEventListener( 'addstream', event => {} );
-peerConnection.addEventListener( 'removestream', event => {} );
+peerConnection.addEventListener( 'track', event => {} );
+peerConnection.addEventListener( 'removetrack', event => {} );
```

Comment:
This event doesn't exist on the peerconnection, it's a stream event.

Reply:
I see. Removed

Reply:
I guess  peer connection's `track` event should be enough for minimal setup. But I can do other events later when I learn the project better.

Reply:
Ahhhhh, no worries 👍🏻
I'll cover that in one of the updates coming unless someone beats me to it haha 😄